deepchem/models/torch_models/scscore.py

```diff
@@ -0,0 +1,165 @@
+    The SCScore model can accurately predict the synthetic complexity of a variety of molecules, including both drug-like and natural product molecules.
+    SCScore has the potential to be a valuable tool for chemists who are working on drug discovery and other areas of chemistry.
+
+    The learned metric (SCScore) exhibits highly desirable nonlinear behavior, particularly in recognizing increases in synthetic complexity throughout a number of linear synthetic routes.
+
+    Our model uses hingeloss instead of the shifted relu loss as in the supplementary material [2]_ provided by the author.
+    This could cause differentiation issues with compounds that are "close" to each other in "complexity".
+
+    The default values for the model are the same as the ones used in the original paper [1]_.
+
+    References
+    ----------
+    .. [1] Coley, C. W., Rogers, L., Green, W., & Jensen, K. F. (2018). "SCScore: Synthetic Complexity Learned from a Reaction Corpus". Journal of Chemical Information and Modeling, 58(2), 252-261. https://doi.org/10.1021/acs.jcim.7b00622
+
+    .. [2] Coley, C. W., Rogers, L., Green, W., & Jensen, K. F. (2018). Supplementary material to "SCScore: Synthetic Complexity Learned from a Reaction Corpus". Journal of Chemical Information and Modeling, 58(2), 252-261. https://github.com/connorcoley/scscore
+    """
+
+    def __init__(self, 
+                 n_features: int=1024, 
+                 layer_sizes: list=[300,300,300,300], 
+                 dropout: float=0.0, 
+                 score_scale: int=5, 
+                 **kwargs):
+        """
+        Parameters
+        ----------
+        n_features: int (default 1024)
+            Number of features per molecule.
+        layer_sizes: list of int (default [300, 300, 300, 300, 300])
+            Size of each hidden layer, length of the list is the number of hidden layers.
+        dropouts: float (default 0.0)
+            Droupout to apply to each hidden layer.
+        score_scale: int (default 5)
+            Max output score, range of output = (1, score_scale).
+        
+        """
+        if dropout < 0.0 or dropout > 1.0:
+            raise ValueError(f"Dropout must be between 0.0 and 1.0, but got {dropout}")
+        
+        if score_scale < 1:
+            raise ValueError(f"Score scale must be greater than 1, but got {score_scale}")
+        
+        model = ScScore(n_features=n_features, 
+                        layer_sizes=layer_sizes, 
+                        dropout=dropout, 
+                        score_scale=score_scale)
+        
+        loss = HingeLoss()
+        
+        super(ScScoreModel, self).__init__(model, 
+                                           loss=loss, 
+                                           **kwargs)
+    
+    def default_generator(self, 
```

Comment:
You can just remove this, inheritance will work out

Reply:
Will do